javatests/arcs/core/data/AccessPathTest.kt

```diff
@@ -62,4 +62,19 @@ class AccessPathTest {
         assertThat("${AccessPath("Reader", connectionSpec, multipleSelectors)}")
             .isEqualTo("hcs:Reader.data.foo.bar")
     }
+
+    @Test
+    fun instantiateForParticle() {
```

Comment:
I think this should be two tests. You've got very clear Arrange, Act, Assert steps, but they're all intermingled.